### DISCUSSION TITLE
alist上传不了大文件

### DISCUSSION THREAD

**Discussion by AbyssSkb** [AlistGo/alist#5269] [Q&A]
如题，当我上传大文件时（超过2GB），传完之后一直卡在正在后台上传的界面，等了很久都没好，小文件没问题

**Comment by 929126387** [comment]
+1

**Comment by SyberRabbit** [comment]
+1

**Reply by SyberRabbit** [reply]
不过，检查了一下文件，确实已经上传完了，视频也可以播放了，就是不知道视频完整性咋样

**Comment by verydemo** [comment]
+1

**Comment by SenkjM** [comment]
上传大文件还是用官方手段靠谱一点，最近回家上传只有50M小水管，加密的临时文件都过期删除了还没上传好。
最后还是用百度自带的文件夹同步把加密后的同步上传了才解决的

**Comment by vayenge** [comment]
我这里是上传大文件会提示成功实际未成功；视频只上传了一部分就提示成功了；损坏视频无法播放；
本地存储的操作
